components/raftstore-v2/tests/integrations/test_basic_write.rs

```diff
@@ -147,4 +147,8 @@ fn test_put_delete() {
     assert!(!resp.get_header().has_error(), "{:?}", resp);
     let snap = router.stale_snapshot(2);
     assert_matches!(snap.get_value(b"key"), Ok(None));
+
+    // Check if WAL is skipped for basic writes.
+    let mut cached = cluster.node(0).tablet_registry().get(2).unwrap();
+    check_skip_wal(cached.latest().unwrap().as_inner().path());
```

Comment:
How does this pass? WAL will be created no matter how according to https://github.com/tikv/rocksdb/blob/8899a360623988adeec1d865bef9d24568ddd32c/db/db_impl/db_impl_open.cc#L1671

Reply:
I check whether the file size is 0 instead of whether the file exists.